.github/workflows/releaseService.yml

```diff
@@ -68,4 +69,4 @@ jobs:
             echo "Publish skipped: missing registry credentials."
           else
             start-sdk publish https://$S9USER:$S9PASS@$S9REGISTRY ${{ env.package_id }}.s9pk
-          fi
+          fi
```

Comment:
seems like maybe your autoformatter might be conflicting with someone else's?

Reply:
Same as above, it looks like the deletion of the trailing whitespace flagged.